README.md

```diff
@@ -150,7 +150,7 @@ Since it is very convenient to use a single file for all of your shortcuts (back
 
 ```
 # this line will launch the notify-send command.
-<Super>h,notify-send Hello world
+<Control><Super>h,notify-send Hello world
```

Comment:
Even though `<Control>` works, the documentation states the `<Ctrl>` and I'd like to stick to single variant. So I'd prefer `<Ctrl>` However, as this is the very first example, would you find another simple and free shortcut, like `<Super>w`, `<Super>e` or something?